app/blueprints/scoring/templates/macros/justification.html

```diff
@@ -1,14 +1,15 @@
 {%- from 'govuk_frontend_jinja/components/textarea/macro.html' import govukTextarea -%}
 
-{% macro justification(form, justification_error) %}
+{% macro justification(form, state, justification_error) %}
```

Comment:
State isn't very specific here, since it's a sub-macro can we name it better?

i.e 
```suggestion
{% macro justification(form, assessment_criteria_link, justification_error) %}
```

Reply:
Changed!